### PR TITLE
LPS-195545 buildLang

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/creator/ObjectEntryInfoItemCreator.java [hunk 1]
```diff
@@ -46,9 +47,11 @@ public ObjectEntryInfoItemCreator(
 
 	@Override
 	public ObjectEntry createFromInfoItemFieldValues(
-			long groupId, InfoItemFieldValues infoItemFieldValues)
+			long groupId, InfoItemFieldValues infoItemFieldValues, int status)
 		throws InfoFormException {
 
+		int objectEntryStatus = status;
```
Comment (ealonso) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/creator/ObjectEntryInfoItemCreator.java:53]: @victorg1991 You should define it near the usage

File: modules/apps/info/info-impl/src/main/java/com/liferay/info/internal/request/struts/EditInfoItemStrutsAction.java [hunk 2]
```diff
@@ -218,14 +219,23 @@ public String execute(
 					throw new InfoFormException();
 				}
 
+				String status = ParamUtil.getString(
+					httpServletRequest, "info-item-status",
+					WorkflowConstants.LABEL_APPROVED);
```
Comment (ealonso) [inline modules/apps/info/info-impl/src/main/java/com/liferay/info/internal/request/struts/EditInfoItemStrutsAction.java]: @victorg1991 we should use STATUS_APPROVED and not labels

File: modules/apps/info/info-impl/src/main/java/com/liferay/info/internal/request/struts/EditInfoItemStrutsAction.java [hunk 3]
```diff
@@ -218,14 +219,23 @@ public String execute(
 					throw new InfoFormException();
 				}
 
+				String status = ParamUtil.getString(
+					httpServletRequest, "info-item-status",
```
Comment (ealonso) [inline modules/apps/info/info-impl/src/main/java/com/liferay/info/internal/request/struts/EditInfoItemStrutsAction.java]: @victorg1991 we should follow portal patterns, and it should be just status and should be an int

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
Baseline error @victorg1991

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/140847

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**